source/administration/incident-response-application.rst

```diff
@@ -0,0 +1,667 @@
+                                    "description": "Close issue as resolved",
+                                    "to": "Resolved"
+                                }
+                            ]
+                        }
+                    ]
+                },
+                {
+                    "name": "Resolved",
+                    "start_actions": [
+                        {
+                            "name": "ResolveWarroom",
+                            "type": "post",
+                            "channel_name": "{{.Action.CreateWarroom.ChannelName}}",
+                            "message": "Resolved."
+                        },
+                        {
+                            "name": "PostResolved",
+                            "type": "post",
+                            "channel_name": "town-square",
+                            "message": "Resolved Security Issue {{.Instance.Number}}"
+                        },
+                        {
+                            "name": "ArchiveSecurityChannel",
+                            "type": "archive_channel",
+                            "channel_name": "security-issue-{{.Instance.Number}}"
+                        }
+                    ]
+                }
+            ]
+        }
+    ]
+
+Statistics
+~~~~~~~~~~~~~~~~~~
+
+The incident response application also enables you to pull summary statistics, including mean-time-to-acknowledgment (MTTA) and mean-time-to-resolution (MTTR).
+
+To pull a sample report, use ``/workflow stats`` in any Mattermost channel:
+
+.. image:: ../images/incident-response-app-statistics.png
+   :alt: Incident Response App: Statistics
+
+The statistics are based on an aggregated summary of all individual workflow instances. You may also reset statistics at any time via ``/workflow reset-stats``.
+
+You must be a System Administrator or an authorized user in **System Console > Plugins > Incident Response App** to execute these commands.
+
+Permissions and Management of the App
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, System Administrators can edit the incident response app by uploading a JSON file via the ``/workflow edit`` command. This allows System Administrators to have full control over what app workflows are configured in a Mattermost server.
+
+You may optionally enable individual users to manage the app by adding a list of authorized user IDs in **System Console > Plugins > Incident Response**. User IDs can be found by navigating to **System Console > User Management**. After clicking into a user's name, their ID is on the right-hand side of the blue header.
+
+Roadmap
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following are some of the use cases we plan to support in a future Beta or stable release:
+
+1. Pulling remote data to, for instance, look up responders who are on duty from an external system or from AD/LDAP, and notifying them about a new incident.
```

Comment:
This is hard to read. Here is an idea, but feel free to change. 
 - Integrating with data in other systems, for example, looking up users in an external system who may be "on-call" and need to be notified of a new event.